test/test_manifest.json

```diff
@@ -4784,5 +4784,12 @@
        "rounds": 2,
        "lastPage": 1,
        "type": "fbf"
-    }
+    },
+    {  "id": "pr12564",
+      "file": "pdfs/pr12564.pdf",
+      "md5": "8033857922eb03d050f6a04c2f5e9851",
+      "rounds": 1,
+      "type": "eq",
+      "annotations": true
+   },
```

Comment:
This comma causes the tests to fail, since it's not valid JSON:
```
### Running unit tests
undefined:4795
]
^

SyntaxError: Unexpected token ] in JSON at position 129429
    at JSON.parse (<anonymous>)
    at getTestManifest (C:\pdfjs\botio-files-pdfjs\private\739c7c0473bff57\test\test.js:329:23)
    at ensurePDFsDownloaded (C:\pdfjs\botio-files-pdfjs\private\739c7c0473bff57\test\test.js:888:18)
    at main (C:\pdfjs\botio-files-pdfjs\private\739c7c0473bff57\test\test.js:918:5)
    at Object.<anonymous> (C:\pdfjs\botio-files-pdfjs\private\739c7c0473bff57\test\test.js:935:1)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

Please make sure that patches pass all tests *locally*, before submitting a patch; please find additional details at https://github.com/mozilla/pdf.js/wiki/Contributing#4-run-lint-and-testing

Reply:
omg, sorry about that. 
Fixed.